tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -538,7 +536,8 @@ object QualificationAppInfo extends Logging {
     val estimated_wall_clock_dur_not_on_gpu = appDuration - speedupOpportunityWallClock
     val estimated_gpu_duration =
       (speedupOpportunityWallClock / speedupFactor) + estimated_wall_clock_dur_not_on_gpu
-    val estimated_gpu_speedup = appDuration / estimated_gpu_duration
+    // force GPU speedup to be 1.0 if the application is not applicable
```

Comment:
I was hoping not to do this as we lose information, how about we put it as -1 to be clear that it was intentionally cleared.

Reply:
Done! changed to -1.